auth/server/src/main/java/com/seagates3/controller/IAMController.java

```diff
@@ -193,6 +175,27 @@ ServerResponse serve(FullHttpRequest httpRequest,
           LOGGER.error("Incorrect signature. Request not authenticated");
           return serverResponse;
         }
+
+      } else {
+
+        try {
+          requestor = RequestorService.getRequestor(clientRequestToken);
+        }
+        catch (InvalidAccessKeyException ex) {
+          LOGGER.debug(ex.getServerResponse().getResponseBody());
+        }
+        catch (InternalServerException ex) {
+          LOGGER.debug(ex.getServerResponse().getResponseBody());
+        }
+        catch (InvalidRequestorException ex) {
+          LOGGER.debug(ex.getServerResponse().getResponseBody());
+        }
+        if (requestor == null) {
+          LOGGER.error("Invalid user, authentication failed");
+          AuthenticationResponseGenerator responseGenerator =
+              new AuthenticationResponseGenerator();
+          serverResponse = responseGenerator.invalidAccessKey();
+          return serverResponse;
       } else {
         LOGGER.debug("Validating user with accesskey and secretkey entered");
```

Comment:
remove these log statements may not be use full

Reply:
removed